src/CONST.js

```diff
@@ -670,7 +670,6 @@ const CONST = {
             FREE: 'free',
             PERSONAL: 'personal',
             CORPORATE: 'corporate',
-            TEAM: 'team',
```

Comment:
@chiragsalian Undoing your revert removed this policy type. Do we actually want this?

Reply:
`POLICY.TYPE.TEAM` isn't on main, so if you pull main I think this will go away. We don't use it anywhere.